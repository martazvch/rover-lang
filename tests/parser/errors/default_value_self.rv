fn bad(self = 5) {}
