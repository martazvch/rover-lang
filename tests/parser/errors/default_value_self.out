default_value_self
